hal/halx86/xbox/reboot.c

```diff
@@ -0,0 +1,104 @@
+/*
+ * PROJECT:         Xbox HAL
+ * LICENSE:         GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:         Xbox reboot functions
+ * COPYRIGHT:       Copyright 2004 Lehner Franz (franz@caos.at)
+ *                  Copyright 2019 Stanislav Motylkov (x86corez@gmail.com)
+ */
+
+/* INCLUDES ******************************************************************/
+
+#include "halxbox.h"
+
+#define NDEBUG
+#include <debug.h>
+
+/* PRIVATE FUNCTIONS *********************************************************/
+
+VOID
+NTAPI
+SMBusWriteByte(UCHAR Address, UCHAR Register, UCHAR Data)
+{
+    INT Retries = 50;
+
+    /* Wait while bus is busy with any master traffic */
+    while (READ_PORT_USHORT((PUSHORT)SMB_GLOBAL_STATUS) & 0x0800);
```

Comment:
There is no define for 0x800 ?
Also, imo it would be cleaner if the wile was layed out like this:
```
while(stuff)
{
    // Do nothing
}
```

Reply:
for the "Do nothing", there is a macro symbol called "NOTHING" (avoids the comment)

Reply:
Fixed while loop style.